lib/takeaway.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'menu'
+require_relative 'user_input'
+require_relative 'order'
+require_relative 'message'
+
+class Takeaway
+
+  attr_reader :menu, :order, :user_input, :message
+
+  def initialize(menu = Menu.new, user_input = UserInput.new,
+    order = Order.new, message = Message.new)
+    @menu = menu
+    @user_input = user_input
+    @order = order
+    @message = message
+  end
+
+  def display_menu
+    menu.display
+  end
+
+  # to be continued: implement a method that manages all the below?
+  # def manage_order
+  #   order_from_menu
+  #   order.total_to_pay
+  #   send_message
+  # end
+
+  def order_from_menu
+    display_menu.each do |item, _|
```

Comment:
I miss knowledge about the _ in the pipe, will read about what it is doing exactly